src/preface.md

```diff
@@ -44,8 +44,8 @@ are a lot of people who do hobby operating system work, but... I don’t like
 their attitudes.
 
 You see, a lot of people see low-level programming as some kind of superior,
-only-for-the-smartest kind of thing. They have a puritanical world-view: I
-suffered to learn this, so you too must suffer, to build character. I think
+only-for-the-smartest kind of thing. They have a puritanical world-view: “I
+suffered to learn this, so you too must suffer to build character”. I think
```

Comment:
Thanks for this! One wrinkle though: period goes inside of double quotes. mind fixing that up?